index.js

```diff
@@ -107,14 +127,14 @@ NYC.prototype.addContent = function (filename, content) {
   }
 }
 
-NYC.prototype.shouldInstrumentFile = function (relFile, returnImmediately) {
-  // only instrument a file if it's not on the exclude list.
-  for (var i = 0, exclude; (exclude = this.exclude[i]) !== undefined; i++) {
-    if (exclude.test(relFile)) {
-      return false
-    }
+NYC.prototype.shouldInstrumentFile = function (filename, relFile) {
+  relFile = relFile.replace(/^\.\//, '') // remove leading './'.
+
+  if (this.include) {
+    return micromatch.any(filename, this.include) || micromatch.any(relFile, this.include)
```

Comment:
Most API's require you to take absolute file names into account when specifying a glob pattern `**/test/**`, I am not sure if there is a good reason behind this or not. 

Might it be better to make people write their excludes with a leading `./`, and then replace that with the dirname of `package.json` when we load the config? So `./test/**` becomes `/Users/benjamincoe/bcoe/yargs/test/**`. It's certainly more explicit. 

I don't have a good reason not to do it this way, other than "it's not how I've seen it done before". Certainly not a good enough reason not to do it, but I am curious why others don't.